app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt

```diff
@@ -269,7 +269,7 @@ private fun SelfUserProfileTopBar(
                 fillMaxWidth = false,
                 minHeight = dimensions().userProfileLogoutBtnHeight,
                 state = WireButtonState.Error,
-                clickBlockParams = ClickBlockParams(blockWhenSyncing = true, blockWhenConnecting = true),
+                clickBlockParams = ClickBlockParams(blockWhenSyncing = false, blockWhenConnecting = false),
```

Comment:
even when syncing ? we might loose some data if it's not processed and we logout before